detect lint errors

Run regal lint --format=github policies samples
Rule:         	defer-assignment                                                        	
Level:        	error                                                                   	
Description:  	Assignment can be deferred                                              	
Category:     	performance                                                             	
Location:     	policies/rbac/backstage/global/result_test.rego:92:2                    	
Text:         	decision1 := global.conditional("catalog", "catalog-entity", condition1)	
Documentation:	https://docs.styra.com/regal/rules/performance/defer-assignment         	
              	
Rule:         	defer-assignment                                                        	
Level:        	error                                                                   	
Description:  	Assignment can be deferred                                              	
Category:     	performance                                                             	
Location:     	policies/rbac/backstage/global/result_test.rego:118:2                   	
Text:         	decision1 := global.conditional("catalog", "catalog-entity", condition1)	
Documentation:	https://docs.styra.com/regal/rules/performance/defer-assignment         	
              	
Rule:         	defer-assignment                                                        	
Level:        	error                                                                   	
Description:  	Assignment can be deferred                                              	
Category:     	performance                                                             	
Location:     	policies/rbac/backstage/global/result_test.rego:169:2                   	
Text:         	decision1 := global.conditional("catalog", "catalog-entity", condition1)	
Documentation:	https://docs.styra.com/regal/rules/performance/defer-assignment         	

[4](https://github.com/ap-communications/platt-policy-template/actions/runs/12060705211/job/33631567943?pr=18#step:8:5)1 files linted. 3 violations found in 1 file.
Error: Assignment can be deferred. To learn more, see: https://docs.styra.com/regal/rules/performance/defer-assignment
Error: Assignment can be deferred. To learn more, see: https://docs.styra.com/regal/rules/performance/defer-assignment
Error: Assignment can be deferred. To learn more, see: https://docs.styra.com/regal/rules/performance/defer-assignment
Error: Process completed with exit code 3.

--- a/policies/rbac/backstage/global/result_test.rego
+++ b/policies/rbac/backstage/global/result_test.rego
@@ -89,12 +89,12 @@ test_aggreate_conditional_decisions_multiple if {
 		"rule": "HAS_LABEL",
 		"params": {"label": "test-label"},
 	}
-	decision1 := global.conditional("catalog", "catalog-entity", condition1)
 	condition2 = {
 		"resourceType": "catalog-entity",
 		"rule": "IS_ENTITY_OWNER",
 		"params": {"claims": ["test-claim-1", "test-claim-2"]},
 	}
+	decision1 := global.conditional("catalog", "catalog-entity", condition1)
 	decision2 := global.conditional("catalog", "catalog-entity", condition2)
 	results := [
 		{"decision": {"result": "ALLOW"}},
@@ -115,12 +115,12 @@ test_aggreate_conditional_decisions_multiple_with_dups if {
 		"rule": "HAS_LABEL",
 		"params": {"label": "test-label"},
 	}
-	decision1 := global.conditional("catalog", "catalog-entity", condition1)
 	condition2 = {
 		"resourceType": "catalog-entity",
 		"rule": "IS_ENTITY_OWNER",
 		"params": {"claims": ["test-claim-1", "test-claim-2"]},
 	}
+	decision1 := global.conditional("catalog", "catalog-entity", condition1)
 	decision2 := global.conditional("catalog", "catalog-entity", condition2)
 	results := [
 		{"decision": {"result": "ALLOW"}},
@@ -166,12 +166,12 @@ test_filter_conditional_decisions_contains_multiple if {
 		"rule": "HAS_LABEL",
 		"params": {"label": "test-label"},
 	}
-	decision1 := global.conditional("catalog", "catalog-entity", condition1)
 	condition2 = {
 		"resourceType": "catalog-entity",
 		"rule": "IS_ENTITY_OWNER",
 		"params": {"claims": ["test-claim-1", "test-claim-2"]},
 	}
+	decision1 := global.conditional("catalog", "catalog-entity", condition1)
 	decision2 := global.conditional("catalog", "catalog-entity", condition2)
 	results := [
 		{"decision": {"result": "ALLOW"}},

